### DISCUSSION TITLE
评论点赞

### DISCUSSION THREAD

**Discussion by alexjjzc** [walinejs/waline#1092] [Q&A] [answered]
我再我的博客部署waline评论系统后，发现评论框中不显示“gif”，同时评论区也没有点赞按钮，请问这两个怎么设置呢？（我的博客采用的是fluid样式）

**Comment by lizheming** [comment, marked as answer]
升级一下主题更新下 Waline 的版本，fluid 已经适配了 Waline 2 啦
